src/Umbraco.Core/StringUdi.cs

```diff
@@ -14,7 +14,7 @@ public class StringUdi : Udi
     /// <param name="entityType">The entity type part of the udi.</param>
     /// <param name="id">The string id part of the udi.</param>
     public StringUdi(string entityType, string id)
-        : base(entityType, "umb://" + entityType + "/" + EscapeUriString(id)) =>
+        : base(entityType, Constants.Conventions.Udi.StartUdi + entityType + "/" + EscapeUriString(id)) =>
```

Comment:
Would string formatting like `$"{Constants.Conventions.Udi.StartUdi}{entityType}/{EscapeUriString(id)}` make it any better?

Reply:
Added a benchmark for string formatting like that. It is interesting that it allocates the same size as the Span function, but is a bit slower (but it is nanoseconds, so it doesn't really matter).

Reply:
Cool that you checked it, it's indeed interesting to know!